### DISCUSSION TITLE
oauth2 redirect_url stuck at 127.0.0.1 ?

### DISCUSSION THREAD

**Discussion by tournesolo** [pocketbase/pocketbase#4351] [Q&A] [answered]
hi -- I seem to be stuck on this apparently trivial issue. 

I have a pocketbase instance which correctly authenticates with two oauth2 providers, when it runs in dev mode. However, when I push it to prod, the redirect_uri stays stuck at `redirect_uri=http%3A%2F%2F127.0.0.1%3A8511%2Fapi%2Foauth2-redirect` although it should be (the escaped version of) `https://example.com/app/pb/api/oauth2-redirect`. 

The `application url` is set to `https://example.com/app/pb` in pocketbase's `settings > application`. And in prod, pocketbase is behind nginx, at 127.0.0.1:myport . 

Is there anything I can do to tell pocketbase to change the redirect_uri away from `127.0.0.1` and to  `https://example.com/app/pb` , aside from setting the  `application url` in `settings > application` ?

Thanks!

**Comment by ganigeorgiev** [comment, marked as answer]
The `authWithOAuth2` redirect url is generated based on the url you use to initialize the SDK (eg. `new PocketBase("https://example.com")`)

**Comment by tournesolo** [comment]
ha, obviously -- I should have thought of that. Thanks for the (super speedy) answer! (And sorry for the distraction.)
